R/db-roracle.R

```diff
@@ -0,0 +1,53 @@
+#' @export
+sql_translate_env.OraConnection <- function(con) {
+  sql_translate_env.Oracle(con)
+}
+
+#' @export
+sql_select.OraConnection <- function(con, select, from, where = NULL,
+                             group_by = NULL, having = NULL,
+                             order_by = NULL,
+                             limit = NULL,
+                             distinct = FALSE,
+                             ...) {
+  sql_select.Oracle(con, select, from, where = where,
+                               group_by = group_by, having = having,
+                               order_by = order_by,
+                               limit = limit,
+                               distinct = distinct)
+}
+
+#' @export
+db_analyze.OraConnection <- function(con, table, ...) {
+  db_analyze.Oracle(con = con, table = table)
```

Comment:
Can you please pass `...` down through all these functions? That keeps it forward compatible